packages/cra-template/package.json

```diff
@@ -5,44 +5,43 @@
   "dependencies": {
     "@bitcoin-computer/components": "^0.17.2-beta.0",
     "@bitcoin-computer/lib": "^0.17.2-beta.0",
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^13.4.0",
-    "@testing-library/user-event": "^13.5.0",
-    "@types/bitcoin-computer__lib": "^0.17.1",
-    "@types/jest": "^27.5.2",
-    "@types/node": "^16.18.16",
-    "@types/react": "^18.0.28",
-    "@types/react-dom": "^18.0.11",
-    "flowbite": "^2.2.1",
+    "@testing-library/jest-dom": "^6.4.2",
+    "@testing-library/react": "^14.2.1",
+    "@testing-library/user-event": "^14.5.2",
+    "@types/bitcoin-computer__lib": "^0.17.2",
```

Comment:
Same here, we should remove this devDependency to see if this change effectively work